deltachat-ios/AppDelegate.swift

```diff
@@ -458,23 +468,30 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
                 return
             }
 
+            self.pushToDebugArray("s4")
             if !self.appIsInForeground() {
+                self.pushToDebugArray("s5:"+String(format: "%.3fs", Double(Date().timeIntervalSince1970)-nowTimestamp))
                 self.dcAccounts.stopIo()
```

Comment:
by some measurements, the time is not wasted here (as i suspected) but somewhere before:
<img width=320 src=https://user-images.githubusercontent.com/9800740/163693393-9ebbc8d1-e288-4405-918b-75bbbfb07fce.PNG>